Week0/README.md

```diff
@@ -1,13 +0,0 @@
-# Reading material for your first JavaScript lesson:
```

Comment:
The link to this README from the main README is broken.

In addition the link to JavaScript Fundamentals README as the bottom of the main README is broken.

Reply:
yes, that is correct, I removed the file and forgot to remove Week0 in the planning